src/styles/blocks/stopwatch.scss

```
@@ -0,0 +1,51 @@
+.stopwatch {
+  position: relative;
+
+  @include watch-square;
+
+  background-image: url(../images/clock-old.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+
+  transition: all 0.3s ease;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```